### PR TITLE
Update run-live-response.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the response to 201 created.
Updated the Limitation where you cannot have more than one API calls to run different live response commands for the same device.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5908

File: microsoft-365/security/defender-endpoint/run-live-response.md [hunk 1]
```diff
@@ -54,7 +54,9 @@ Runs a sequence of live response commands on a device
 
 4.  RunScript command timeouts after 10 minutes.
 
-5.  When a live response command fails all followed actions will not be
+5.  Live response commands cannot be queued up and can only be executed one at a time. 
+
+6.  Multiple Live response commands can be run on a single API call. However, When a live response command fails all followed actions will not be
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/run-live-response.md]:
```suggestion
6.  Multiple live response commands can be run on a single API call. However, when a live response command fails all the following actions will not be
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msbemba, kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**